ee/clickhouse/queries/test/test_trends.py

```diff
@@ -1561,3 +1561,59 @@ def test_same_day(self):
             self.team,
         )
         self.assertEqual(response[0]["data"], [1.0])
+
+    @test_with_materialized_columns(event_properties=["email", "name"], person_properties=["email", "name"])
+    def test_ilike_regression_with_current_clickhouse_version(self):
```

Comment:
Q: is there a potential simpler repo in property.py tests?

Not urgent, but this is hard to follow